.github/workflows/fix-conflicts.yml

```diff
@@ -0,0 +1,27 @@
+name: Solve conflicts
+
+on:
+  pull_request:
+    types:
+      - synchronize
+
+jobs:
+  fix-conflicts:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Resolves merge conflicts
+        if: github.event.label.name == 'has conflicts'
```

Comment:
so this label is also added by a github workflow / action. What if that workflow runs after this new workflow you added? is this what that `syncronize` thing above is for?

Reply:
The ideia is that this workflow should run each time that a PR is update with a push request, but the step that will solve conflicts will run only if a conflict in any of these two files was found